tests/test_cyhy_config.py

```diff
@@ -0,0 +1,213 @@
+    with patch.dict(os.environ, {"CYHY_CONFIG_PATH": "/mock/env/path"}):
+        with patch("cyhy_config.cyhy_config.Path.exists", return_value=True):
+            assert find_config_file() == Path("/mock/env/path")
+
+
+def test_find_config_file_env_var_set_but_does_not_exist():
+    """Test find_config_file when the CYHY_CONFIG_PATH environment variable is set but does not exist."""
+    with patch.dict(os.environ, {"CYHY_CONFIG_PATH": "/mock/env/path"}):
+        with patch("cyhy_config.cyhy_config.Path.exists", side_effect=[False, True]):
+            assert find_config_file() == Path("cyhy.toml")
+
+
+def test_find_config_file_in_current_directory():
+    """Test find_config_file when the cyhy.toml file exists in the current directory."""
+    with patch("cyhy_config.cyhy_config.Path.exists", side_effect=[True]):
+        assert find_config_file() == Path("cyhy.toml")
+
+
+def test_find_config_file_in_home_directory():
+    """Test find_config_file when the cyhy.toml file exists in the user's home directory."""
+    with patch("cyhy_config.cyhy_config.Path.exists", side_effect=[False, True]):
+        assert find_config_file() == Path.home() / ".cyhy/cyhy.toml"
+
+
+def test_find_config_file_in_etc_directory():
+    """Test find_config_file when the cyhy.toml file exists in the /etc directory."""
+    with patch("cyhy_config.cyhy_config.Path.exists", side_effect=[False, False, True]):
+        assert find_config_file() == Path("/etc/cyhy.toml")
+
+
+def test_find_config_file_no_valid_path():
+    """Test find_config_file when no valid path is found."""
+    with patch("cyhy_config.cyhy_config.Path.exists", return_value=False):
+        with pytest.raises(FileNotFoundError):
+            find_config_file()
+
+
+def test_read_config_ssm_env_var_set():
+    """Test read_config_ssm when the CYHY_CONFIG_SSM_PATH environment variable is set."""
+    mock_ssm_client = MagicMock()
+    mock_ssm_client.get_parameter.return_value = {
+        "Parameter": {"Value": 'key = "value"'}
+    }
+
+    with patch("cyhy_config.cyhy_config.client", return_value=mock_ssm_client):
+        with patch.dict(os.environ, {"CYHY_CONFIG_SSM_PATH": "/mock/ssm/path"}):
+            config = read_config_ssm(model=MockModel)
+            assert config.key == "value"
+
+
+def test_read_config_ssm_parameter_not_found():
+    """Test read_config_ssm when the parameter is not found in AWS SSM."""
+    mock_ssm_client = MagicMock()
+    mock_ssm_client.get_parameter.side_effect = ClientError(
+        {"Error": {"Code": "ParameterNotFound"}}, "get_parameter"
+    )
+
+    with patch("cyhy_config.cyhy_config.client", return_value=mock_ssm_client):
+        with patch.dict(os.environ, {"CYHY_CONFIG_SSM_PATH": "/mock/ssm/bad_path"}):
+            assert read_config_ssm() is None
+
+
+def test_read_config_ssm_other_client_error():
+    """Test read_config_ssm.
+
+    When SSM responds with an error code other than ParameterNotFound.
+    """
```

Comment:
Make this consistent with the other descriptions:
```suggestion
    """Test read_config_ssm when SSM responds with an error code other than ParameterNotFound."""
```

Reply:
See:
- [e59ede9](https://github.com/cisagov/cyhy-config/pull/5/commits/e59ede9e36fbeb1075906e040787cdd8a900fee0)